### PR TITLE
Updated testing on Travis.

### LINKED ISSUE #40: Update Travis CI
Companion issue to https://github.com/Gizra/message/issues/131.

### PR DESCRIPTION
**Pull request by jhedstrom**
- Fixes #40.

### CODE REVIEW

**Comment by jhedstrom** [comment]
The composer install is failing with

```
    - The requested package drupal/message could not be found in any version, there may be a typo in the package name.
```

and similar...not sure we want to update it with custom repos just for testing.

File: composer.json [hunk 1]
```diff
@@ -15,7 +15,8 @@
       "drupal/message_notify": "~8.1.0"
     },
     "require-dev": {
-      "drupal/og": "~8.1.0"
+      "drupal/og": "~8.1.0",
```
Comment (amitaibu) [inline composer.json:24]: OG can be removed for now

**Comment by amitaibu** [comment]
> not sure we want to update it with custom repos just for testing.

Maybe we should add to `composer.json`:

```
+    "repositories": {
+        "drupal": {
+            "type": "composer",
+            "url": "https://packagist.drupal-composer.org"
+        }
+    }
```

File: composer.json (hunk 1 above)
Comment (jhedstrom) [reply composer.json:24]: I think one of the tests actually uses og still.

**Comment by jhedstrom** [comment]
I added that, but wonder if that will have conflicts with drupal.org's own packagist stuff...hopefully not :)

File: composer.json (hunk 1 above)
Comment (amitaibu) [reply composer.json:24]: Oh, cool, I like OG :)

**Pull request closed without merging**